backend/tests/Tests/Interpreter.Tests.fs

```diff
@@ -632,6 +632,24 @@ module Fns =
 
   let tests = testList "Fns" [ Builtin.tests; Package.tests ]
 
+module Statement =
+  let simple = t "()\n true" E.Statements.simple (RT.DInt64(1L))
+
+  let nested =
+    t
+      "myFnThatReturnsUnit()\nmyFnThatReturnsUnit()\nmyFnThatReturnsUnit()\n0L"
+      E.Statements.nested
```

Comment:
The test was something like 
```
Builtin.printLine("1st statement 1st expr")
Builtin.printLine("1st statement 2nd expr")
Builtin.printLine("2nd statement 1st expr")
0L
```
but it created some noise when running tests
example:
![image](https://github.com/user-attachments/assets/febde40b-e0e6-4b77-aea9-9da7e2aa0834)

So, I updated the test values to create a function that returns a Unit. I'm not sure if this is the best thing to do here

https://github.com/darklang/dark/blob/5bb0a03d221c52174ad29201e3fee324c04f9eb3/backend/tests/Tests/TestValues.fs#L499C1-L514C57

https://github.com/darklang/dark/blob/5bb0a03d221c52174ad29201e3fee324c04f9eb3/backend/tests/Tests/TestValues.fs#L670C1-L678C41

Reply:
This seems good to me